src/main/java/com/projectg/geyserupdater/common/util/GeyserApi.java

```diff
@@ -0,0 +1,32 @@
+package com.projectg.geyserupdater.common.util;
+
+import com.google.gson.Gson;
+import com.projectg.geyserupdater.common.pojo.Root;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class GeyserApi {
+    public Root endPoints() {
```

Comment:
1) Should be static and the constructor for this class should be private
2) need a better method name - `fetchRoot` maybe?